### PR TITLE
Add urllib3 Newsletter September 2020

### PR DESCRIPTION
**Pull request by sethmlarson**

### CODE REVIEW

**Review by pquentin — APPROVED** [review]
Thanks! That's a lot indeed :)

**Review by jalopezsilva — APPROVED** [review]
Looks great to me. Awesome work everyone!

**Review by hodbn — APPROVED** [review]
Looks great 👍

File: app/markdown/2020-10-05/urllib3-newsletter-september-2020.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+The major new feature coming in urllib3 `v1.26.0` is being able to
+use HTTPS proxies to tunnel to HTTPS hosts.
+
+Previously this wasn't possible, you would be forced to use HTTP to
+connect to your proxy and then create an HTTPS tunnel to the origin
+through your proxy. This means that the connection between you and
+your proxy wasn't secure and only the connection to the origin was secure.
+
+*Obviously not ideal!* Combine this with HTTPS proxies being more
+common and there's an obvious demand for this functionality.
```
Comment (pquentin) [inline app/markdown/2020-10-05/urllib3-newsletter-september-2020.md]: One use case we can mention is that this allows the proxy to authenticate clients, which is an useful way to replace passwords in a corporate settings. I think this was part of @jalopezsilva's motivation here.

**Review by pquentin — APPROVED** [review]

**Pull request merged**